paasta_tools/setup_marathon_job.py

```diff
@@ -647,8 +648,12 @@ def log_deploy_error(errormsg: str, level: str='event') -> None:
                 new_client.scale_app(app_id=new_app.id, instances=(new_app.instances - slack), force=True)
 
         # TODO: don't take actions in deploy_service.
+        if scaling_app_unhappy_tasks is None:
```

Comment:
I don't think we should do this. 
A task can be unhappy because of the infrastructure *or* the app, if we need to undrain we should undrain.

Reply:
OK, discarding